apt.txt

```diff
@@ -0,0 +1 @@
+openjdk-8-jre
```

Comment:
Is it possible to avoid putting this at the top level of the project? I tried to stuff everything into dev/ as possible.

Reply:
I can place is under `binder` directory, see https://mybinder.readthedocs.io/en/latest/introduction.html

> Configuration files may be placed in the root of your repository or in a `binder/` folder in the repository’s root (i.e. `myproject/binder/`).

I can place it under `binder` directory if that sounds better.

Reply:
@srowen and @holdenk, just to clarify, `spark/dev/binder` isn't possible. `spark/binder` is possible. Do you prefer this? I don't mind doing it.